gdk/static/cli_model.json

```diff
@@ -103,7 +103,7 @@
                     "-o",
                     "--options"
                 ],
-                "help": "Extra configuration options used during component version creation. This argument needs to be a valid json string or a json file path containing publish options."
+                "help": "Extra configuration options used during component version creation. This argument needs to be a valid json string or file path to a JSON file containing... containing publish options."
```

Comment:
not ..., I just put that to say to keep the rest of the text.

containing is duplicated too

Reply:
That got messed up. Fixed it now. Lol.